actions/views/rhs.ts

```diff
@@ -390,6 +391,34 @@ export function selectPost(post: Post) {
     };
 }
 
+export function highlightReply(post: Post) {
+    return {
+        type: ActionTypes.HIGHLIGHT_REPLY,
+        postId: post.id,
+    };
+}
+
+export const clearHighlightReply = {
+    type: ActionTypes.CLEAR_HIGHLIGHT_REPLY,
+};
+
+export const debouncedClearHighlightReply = debounce((dispatch) => {
```

Comment:
Interesting. I don't think I've seen debounce used with the trailing option set. This works similar to using setTimeout every time this is called and cancelling the previous timeout if it hadn't occurred yet, right?

Reply:
yes it's exactly the same